_people-use-web/user-stories.es.md

```diff
@@ -0,0 +1,403 @@
+-   [Mensajes de error y de éxito útiles (interacción)](/people-use-web/tools-techniques/#interaction)
+-   [Navegación con teclado (interacción)](/people-use-web/tools-techniques/#interaction)
+-   [Enlaces de salto (interacción)](/people-use-web/tools-techniques/#interaction)
+
+**Principios de accesibilidad:**
+
+-   [Alternativas textuales para contenido no textual (Perceptible)](/fundamentals/accessibility-principles/#alternatives)
+-   [Subtítulos y otras alternativas para multimedia (Perceptible)](/fundamentals/accessibility-principles/#captions)
+-   [El contenido se puede presentar de diferentes formas (Perceptible)](/fundamentals/accessibility-principles/#adaptable)
+-   [El contenido es más fácil de ver y de oír (Perceptible)](/fundamentals/accessibility-principles/#distinguishable)
+-   [Funcionalidad disponible desde el teclado (Operable)](/fundamentals/accessibility-principles/#keyboard)
+-   [Los usuarios pueden navegar fácilmente, encontrar contenido y determinar dónde se encuentran (Operable)](/fundamentals/accessibility-principles/#navigable)
+-   [El contenido es legible y comprensible (Comprensible)](/fundamentals/accessibility-principles/#readable)
+-   [El contenido se muestra y se opera de forma predecible (Comprensible)](/fundamentals/accessibility-principles/#predictable)
+-   [El contenido es compatible con herramientas presentes y futuras (Robusto)](/fundamentals/accessibility-principles/#compatible)
+
+{% include_cached excol.html type="end" %}
+
+## Preety, estudiante de secundaria con trastorno por déficit de atención e hiperactividad y dislexia {#classroomstudent}
+
+Preety es una estudiante de secundaria con trastorno por déficit de atención e hiperactividad y dislexia. Aunque tiene dificultades considerables para leer, disfruta particularmente en las clases de literatura.
+
+En la escuela de Preety se han empezado a usar libros electrónicos recientemente. Al principio, Preety estaba preocupada por el nuevo formato, pero gracias a los programas de texto a voz que resaltan el texto en la pantalla mientras lo leen en voz alta, se ha dado cuenta de que puede concentrarse en el contenido más fácilmente en lugar de tener que pelar con cada palabra. El programa de texto a voz también sirve de ayuda con otros textos en línea, pero la experiencia de Preety con los sitios web varía mucho de un sitio a otro. Algunos sitios usan gráficos e ilustraciones de forma que le ayudan a concentrarse en las secciones y en el contenido que le gustaría leer, mientras que otros sitios tienen muchos anuncios y contenido que se mueve que provocan distracciones. También encuentra problemas con el contenido en línea cuando la navegación no es muy evidente. Le parece que son más fáciles de usar los sitios web y aplicaciones que proporcionan diferentes formas de navegar, como por ejemplo barra de navegación, herramienta de búsqueda, migas de pan y un mapa del sitio.
```

Comment:
s/que pelar con/que pelear con

Reply:
Awesome, great catch. That's an eye for detail.